rimraf.js

```diff
@@ -38,6 +38,7 @@ function defaults (options) {
   }
   options.disableGlob = options.disableGlob || false
   options.glob = options.glob || defaultGlobOpts
+  options.glob.absolute = true
```

Comment:
Though this line breaks the original object, [the object is already mutated elsewhere](https://github.com/isaacs/rimraf/issues/132). So I don't care about mutation in this pull request.